resources/deletedb.yaml

```diff
@@ -1,3 +1,45 @@
+---
```

Comment:
I could "group them a little" though, to reduce the number of different "sa" needed